packages/components/src/internal/components/assay/AssayImportPanels.tsx

```diff
@@ -236,28 +237,31 @@ class AssayImportPanelsBody extends Component<Props, State> {
             workflowTask = isNaN(_workflowTask) ? undefined : _workflowTask;
         }
 
-        getQueryDetails(schemaQuery).then(queryInfo => {
-            const sampleColumnData = assayDefinition.getSampleColumn();
-            this.setState(
-                () => ({
-                    model: new AssayWizardModel({
-                        // Initialization is done if the assay does not have a sample column and we aren't getting the
-                        // run properties to show for reimport
-                        isInit: sampleColumnData === undefined && this.runAndBatchPropsLoaded(),
-                        assayDef: assayDefinition,
-                        batchColumns: assayDefinition.getDomainColumns(AssayDomainTypes.BATCH),
-                        runColumns: assayDefinition.getDomainColumns(AssayDomainTypes.RUN),
-                        runId,
-                        usePreviousRunFile: this.isReimport(),
-                        batchProperties: this.getBatchPropertiesMap(),
-                        runProperties: this.getRunPropertiesMap(),
-                        queryInfo,
-                        workflowTask,
-                    }),
+        const batchQueryInfo = await getQueryDetails(new SchemaQuery(assayDefinition.protocolSchemaName, 'Batches'));
```

Comment:
Does LKSM support batch fields?

Reply:
Yes...sort of. We don't allow adding batch props through the LKSM app UI for a newly created assay design. But if you have an assay design created in LKS that already has batch props, we do support it in the LKSM assay UI.